.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,222 @@
+1. Under the section "Environment Variables", type `GITHUB_TOKEN` in the "Name"
+text box and paste your personal access token into the "Value" text box.
+1. The `deploy` Travis commands found in [`.travis.yml`](.travis.yml) are then
+used to access this GitHub PAT.
+
+## Workflow for Adding Content
+
+- Use the `book` branch as the main "development" branch (**not `master`**).
+- Since `book` is "protected", to add to the repo you must:
+    1. Submit a PR of a branch from their fork of the repo
+    1. Each PR must pass the Travis CI check and must get at least one (1)
+    approval from someone else.
+    1. Generally @gvwilson or @lwjohnst86 merge into the main repo's
+    `book` branch to make sure the content will generate correctly to the website.
+    However, others should be able to merge as well.
+    
+**For larger changes (e.g. making a chapter)**:
+
+- Pick a chapter (as assigned to or by you) and create a branch to work on for
+that chapter.
+- Make some edits and changes.
+- Create a **[Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)** to
+the main repo.
+- Add the label "work in progress" to that PR.
+- When it's ready for review, remove the label, change the PR status to "Ready
+to review", and post a note in Slack asking for a reviewer.
+
+**For other changes**:
+
+- If you are making a smaller change, please create a branch with a meaningful
+name, submit changes, and ask for a reviewer. 
+- Finally, if you are doing a major reorganization that involves multiple
+chapters, please put in a proposal first.
+
+## Reviewing PRs
+
+- Add yourself as a reviewer to the PR.
+- If making a suggested change to the actual text, please use the
+["Insert a suggestion"](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request)
+    - For the **"reviewee"**, please accept those changes. GitHub will insert the
+    suggestion directly as a commit, thus giving attribution to the reviewer. Note:
+    This does not work for multi-line suggestions edits (though you can add more
+    lines in the suggestion if desired).
+- For individual chapter reviews, look for consistency, content, clarity of
+material with exercises. Don't worry too much about grammar appearance, structure,
+etc. This will be done at the end stage.
+
+## Specific Content Guidelines
+
+1.  bookdown's behavior is controlled by the metadata in `_bookdown.yml` and in the YAML header of `index.Rmd`.
+    Please do not add YAML headers to other `.Rmd` files, since it is ignored when building the entire book.
+
+1.  `_bookdown.yml` contains an ordered list of R Markdown chapters and appendices.
+    This list is repeated in `Makefile`.
+
+1.  The first element of each chapter must be a level-1 heading.
+    Anything before this heading is silently ignored.
+
+1.  The ID for each chapter and section heading is set by putting `{#some-label}` on the heading line.
```

Comment:
Is bookdown not using [Pandoc's default with auto_identifiers](https://pandoc.org/MANUAL.html#extension-auto_identifiers) to generate these automatically?

Reply:
Or do we do this to ensure no duplicates? If so, should we preface with the course (py/r/rse)?

Reply:
This text was there from @gvwilson. bookdown does use pandoc auto identifying, but I think in this case it's to help with duplicates and to distinguish course sections, as you say. I'll add the "py", "r", etc comment.